compiler/testData/diagnostics/tests/platformTypes/nullabilityWarnings/delegatedProperties.fir.kt

```diff
@@ -19,5 +19,5 @@ public class J {
 // FILE: k.kt
 
 var A by J.staticNN
-var B by J<!UNSAFE_CALL!>.<!>staticN
+var B by <!DELEGATE_SPECIAL_FUNCTION_NONE_APPLICABLE!>J.staticN<!>
```

Comment:
Did you know why FE 1.0 reports two diagnostics here? https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/platformTypes/nullabilityWarnings/delegatedProperties.kt#L22 Do we miss something?

Reply:
Nice catch! It's because we are missing the check for `setValue`, which is caused by a missing source on the calleeExpression when building the property accessor. I will fix it.